### ISSUE TITLE
It is available to design the loss function by myself? If so, how can i change the loss function?

### ISSUE THREAD

**Issue by Jimmy880** [shenweichen/DeepCTR#44]
**Describe the question(问题描述)**
English:
In order to deal with the imbalance sample question, I would like to change the loss function to the `tf.nn.weighted_cross_entropy_with_logits` and set `pos_weight` parameter. However, I don't know how to change it in your framework?

Actually, I have noticed that you define the loss function through `model.compile() ` function. But I don't know if this method works for my problem.

中文：
我想知道怎么替换你框架中的loss函数，我想用 `tf.nn.weighted_cross_entropy_with_logits` 这个函数处理样本不平衡的数据，但是不知道如何加入你的框架中

我注意到你在代码中用了`model.compile() `去定义loss函数和优化器，但是不知道这种方式是否对我的问题有用，而且也不知道如何将loss所带的参数通过这个方法传递进去。

**Operating environment(运行环境):**
 - python version 3.6
 - tensorflow version 1.12.0
 - deepctr version 0.3.1

**Comment by shenweichen** [comment]
Hi, @Jimmy880 
You can use the `class_weight` arg in fit method.
for example,if we want to set the **pos_weight=1.5 and neg_weight=1**,then

```python
model.fit(x,y,class_weight={0:1.0,1:1.5})
```

**Comment by Jimmy880** [comment]
@shenweichen 
Thanks for your reply. It really helps me a lot in training imbalance dataset.

I wonder whether the `class_weight` parameter should be passed in the dict format. I used xgboost and sklearn before and in their lib the weight of binary class can be passed in a ratio as `negative sample number/ positive sample number`. Could it also work in here?
